### PR TITLE
Edit Component

### PR DESCRIPTION
**Pull request by RandSohail**
- Open modal that allows the user to 
- Edit the title of the work 
- Edit the content
- Edit the image

### CODE REVIEW

**Review by abdulrahman-2020 — APPROVED** [review]

**Review by AbdallahGot — APPROVED** [review]

File: client/src/Components/AddWorkButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        });
+    } catch (error: any) {
+      message.error(t('error-delete-message'));
+    }
+  };
+  const normFile = (e: any) => {
+    if (Array.isArray(e)) return e;
+    return e && e.fileList;
+  };
+  const checkImage = (file: any) => {
```
Comment (devstm) [inline client/src/Components/AddWorkButton/index.tsx:36]: try to user accept props to Upload Component to allow to user upload specific files type 
in ant design: 
image

and if you jump to See [input accept Attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept)

`<input type="file" id="docpicker"
  accept=".doc,.docx,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document">`

File: client/src/Components/AddWorkButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  const { t } = useTranslation();
+  const [isClickAddWork, setIsClickAddWork] = useState(false);
+  const [form] = Form.useForm();
+  const { TextArea } = Input;
+
+  const handelAddWork = () => {
+    try {
+      form
+        .validateFields()
+        .then(async (values) => {
```
Comment (devstm) [inline client/src/Components/AddWorkButton/index.tsx:19]: then and await? 
what is this?

File: server/controllers/errors/server.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // eslint-disable-next-line no-unused-vars
 const serverError = (err, req, res, next) => {
+  console.log(err);
```
Comment (devstm) [inline server/controllers/errors/server.js:3]: remove the console hhhhhhhhhhhhhhhhhhaaaaaa

**Review by devstm — APPROVED** [review]

File: server/utils/s3.js [hunk 4]
```diff
@@ -18,12 +17,13 @@ const s3 = new S3({
...
   const extension = path.extname(img.name);
   const uploadParams = {
     Bucket: bucketName,
-    Body: fileStream,
+    Body: base64Data,
     Key: `${userId}/${uuid.v1()}${extension}`,
+    ContentEncoding: 'base64',
   };
   return s3.upload(uploadParams).promise();
 }
```
Comment (devstm) [inline server/utils/s3.js:29]: nothing

**Review by devstm — CHANGES_REQUESTED** [review]

**Review by abdulrahman-2020 — APPROVED** [review]

**Review by devstm — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**